app/src/main/scala/org/alephium/explorer/persistence/dao/BlockDao.scala

```diff
@@ -191,8 +191,10 @@ object BlockDao {
   }
 
   def updateInputs(inputs: Seq[InputEntity])(implicit ec: ExecutionContext,
-                                             dc: DatabaseConfig[PostgresProfile]): Future[Int] =
+                                             dc: DatabaseConfig[PostgresProfile]): Future[Unit] =
```

Comment:
if I remember correctly, having the number of updates doesn't bring much, because there is now different kind of updates: tx per address, tokens etc that happen.